### PR TITLE
tgui-next: Turbine computer and station alert console

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
Slight backend changes in turbine to cover a few disconnection cases. Alternates between power/temp/rpm status and connection status depending on if the turbine is there and in working order or not.
image

Station alert pretty much the same as before.
image

@actioninja @stylemistake  Nitpick away please! Station alert has a lot of repeated code, I couldnt figure out how to name the section from the alert type.

## Why It's Good For The Game
tgui bad

## Changelog
:cl: Skoglol
add: tgui-next turbine computer and station alert console
/:cl:

### CODE REVIEW

**Review by stylemistake — APPROVED** [review]

**Comment by stylemistake** [comment]
Only thing I don't like in general, is that everyone likes to wrap `.map()` with a length check.

You can save 1 line of code and 1 indentation level by writing it like this:

```
{fire.length === 0 && (
  <li className="color-good">
    System Nominal
  </li>
)}
{fire.map(alert => (
  <li key={alert}>
    {alert}
  </li>
))}
```

That was my only nitpick. :)

File: tgui-next/packages/tgui/interfaces/StationAlertConsole.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <Fragment>
+      <Section title="Fire Alarms">
+        <ul>
+          {fire.length === 0 && (
+            <li className="color-good">
+              Systems Nominal
+            </li>
+          )}
+          {fire.map(alert => (
+            <li key={alert}>
```
Comment (AnturK) [inline tgui-next/packages/tgui/interfaces/StationAlertConsole.js]: Shouldn't these have some color ?

Comment (kriskog) [reply tgui-next/packages/tgui/interfaces/StationAlertConsole.js]: Nah, they dont. We can add something to make it more clear though
image

Comment (AnturK) [reply tgui-next/packages/tgui/interfaces/StationAlertConsole.js]: Alright, just my imagination then.

**Pull request merged**